tests/unit/bokeh/embed/test_bundle.py

```diff
@@ -67,6 +67,30 @@ def test_mathtext() -> None:
     test_mathtext = TeX()
     return test_mathtext
 
+@pytest.fixture
+def test_mathstring_axis_label() -> None:
+    from bokeh.models import LinearAxis
+    test_mathstring = LinearAxis(axis_label = "$$sin(x)$$")
+    return test_mathstring
+
+@pytest.fixture
+def test_mathstring_axis_label_with_parenthesis() -> None:
+    from bokeh.models import LinearAxis
+    test_mathstring = LinearAxis(axis_label = r"\(sin(x)\)")
+    return test_mathstring
+
+@pytest.fixture
+def test_mathstring_major_label_overrides() -> None:
+    from bokeh.models import LinearAxis
+    test_mathstring = LinearAxis(major_label_overrides = {0: r"\[sin(x)\]"})
+    return test_mathstring
+
+@pytest.fixture
+def test_plaintext() -> None:
+    from bokeh.models import PlainText
+    test_plaintext = PlainText("$$sin(x)$$")
+    return test_plaintext
```

Comment:
What is the advantage of making these fixtures? They seem like they could just be module-scope vars defined at the top of the test module (which is fine)

Reply:
I just followed the current design of the file to be honest.